common/api/summary/ui-framework.exports.csv

```diff
@@ -678,6 +679,8 @@ beta;UserSettingsProvider
 public;UserSettingsStorage 
 internal;useSavedFrontstageState(frontstageDef: FrontstageDef): void
 internal;useSaveFrontstageSettings(frontstageDef: FrontstageDef): void
+public;useScheduleAnimationDataProvider(viewport: ScreenViewport | undefined): ScheduleAnimationTimelineDataProvider | undefined
+beta;useSolarDataProvider(viewport: ScreenViewport | undefined): SolarDataProvider | undefined
```

Comment:
Why is the solar one beta?

Reply:
We're not ready with the solar one yet -- we know of at least one problem that requires a refactor. Bill and I are discussing what to do with it.